_config.yml

```diff
@@ -22,7 +22,7 @@ keep_files: [vendor/assets, docs/master/_static/js/vendor/]
 github: [metadata]
 external_urls:
   github: https://github.com/pytorch/pytorch
-  github_issues: https://github.com/pytorch/pytorch/issues
+  issues: https://github.com/pytorch/pytorch/issues
```

Comment:
Bruce said we should use this link for the hub issues: https://github.com/pytorch/hub/issues. You can probably just name it `hub_issues`.